README.md

```diff
@@ -1,12 +1,299 @@
+| producer.compression.codec                     | gzip          | string                                                             | CRON_PRODUCER_COMPRESSION_CODE                      |
+| producer.retry.backoff.ms                      | 200           | integer                                                            | CRON_PRODUCER_RETRY_BACKOFF_MS                      |
+| producer.message.send.max.retries              | 10            | integer                                                            | CRON_PRODUCER_MESSAGE_SEND_MAX_RETRIES              |
+| producer.socket.keepalive.enable               | true          | boolean                                                            | CRON_PRODUCER_SOCKET_KEEPALIVE_ENABLE               |
+| producer.queue.buffering.max.messages          | 100000        | integer                                                            | CRON_PRODUCER_QUEUE_BUFFERING_MAX_MESSAGES          |
+| producer.queue.buffering.max.ms                | 1000          | integer                                                            | CRON_PRODUCER_QUEUE_BUFFERING_MAX_MS                |
+| producer.batch.num.messages                    | 1000000       | integer                                                            | CRON_PRODUCER_BATCH_NUM_MESSAGES                    |
+| producer.dr_cb                                 | true          | boolean                                                            | CRON_PRODUCER_DR_CB                                 |
+| producer.enable.idempotence                    | false         | boolean                                                            | CRON_PRODUCER_ENABLE_IDEMPOTENCE                    |
+| producer.max.in.flight.requests.per.connection | 1000000       | integer                                                            | CRON_PRODUCER_MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION |
+
+#### Consumer
+
+##### **Main object**
+
+| Key                                       | Default Value | Valid Values | Environment variable                   |
+| ----------------------------------------- | ------------- | ------------ | -------------------------------------- |
+| sdkConsumer.in.processing.max.messages    | 1             | integer      | CRON_SDK_IN_PROCESSING_MAX_MESSAGES    |
+| sdkConsumer.queued.max.messages.bytes     | 10485760      | integer      | CRON_SDK_QUEUED_MAX_MESSAGES_BYTES     |
+| sdkConsumer.subscription.backoff.min.ms   | 1000          | integer      | CRON_SDK_SUBSCRIPTION_BACKOFF_MIN_MS   |
+| sdkConsumer.subscription.backoff.max.ms   | 60000         | integer      | CRON_SDK_SUBSCRIPTION_BACKOFF_MAX_MS   |
+| sdkConsumer.subscription.backoff.delta.ms | 1000          | integer      | CRON_SDK_SUBSCRIPTION_BACKOFF_DELTA_MS |
+| sdkConsumer.commit.interval.ms            | 5000          | integer      | CRON_SDK_COMMIT_INTERVAL_MS            |
```

Comment:
Maybe to keep the pattern and avoid some errors of using configManager lib, you can copy the labels names  from  v2k, k2v, kafka-ws and influxdb-storer